src/traits.rs

```diff
@@ -375,6 +375,20 @@ pub trait ProcessExt: Debug {
     /// ```
     fn cpu_usage(&self) -> f32;
 
+    /// Returns the total accumulated CPU usage (in
+    /// CPU-seconds). Notice that it might be bigger than the total
+    /// clock run time of a process if run on a multi-core machine.
+    ///
+    /// ```no_run
+    /// use sysinfo::{Pid, ProcessExt, System, SystemExt};
+    ///
+    /// let s = System::new_all();
+    /// if let Some(process) = s.process(Pid::from(1337)) {
+    ///     println!("{}sec", process.total_cpu_usage());
+    /// }
+    /// ```
+    fn total_cpu_usage(&self) -> f32;
```

Comment:
The naming isn't great. Maybe something like `total_accumulated_cpu_usage` instead?

Reply:
I'm fine with whatever you think is best for the name.

Reply:
Changed in cf86f66

Reply:
:+1: